apps/openssl.c

```diff
@@ -296,12 +296,28 @@ static void list_md_fn(const EVP_MD *m,
     }
 }
 
+static void list_missing_help(void)
+{
+    const FUNCTION *fp;
+    const OPTIONS *o;
+
+    for (fp = functions; fp->name != NULL; fp++) {
+        if ((o = fp->help) == NULL)
+            continue;
+        for ( ; o->name != NULL; o++) {
+            if (o->helpstr == NULL)
```

Comment:
is formatting like this better:

for (int pflag = 0 ; o->name != NULL; o++) {
         if (o->helpstr == NULL) {
                 if (!pflag) {
                         BIO_printf(bio_out, "%s:\n", fp->name);
                         pflag = 1;
                 }       
                 BIO_printf(bio_out, "\t %s\n", o->name);
         }       
}